cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,37 @@
 /// <reference types='cypress' />
 
+import { user } from "../support/testData";
 describe('Book Store app', () => {
   before(() => {
-    
+    cy.visit('/login');
   });
 
-  it('', () => {
-    
+  it('should provide an abbility to login', () => {
+    cy.get('#userName').type(user.username);
+    cy.get('#password').type(user.password);
+    cy.get('#login').click();
+    cy.url().should('include', '/profile');
   });
+  it('should providean abbility to add a book to collection', () => {
```

Comment:
There's a typo in the description of the test, it should be 'should provide an ability to add a book to collection'.